test/general/TestLocations.py

```diff
@@ -0,0 +1,16 @@
+import unittest
+from collections import Counter
+from worlds.AutoWorld import AutoWorldRegister
+from . import setup_default_world
+
+
+class TestBase(unittest.TestCase):
+    def testCreateDuplicateLocations(self):
+        for game_name, world_type in AutoWorldRegister.world_types.items():
+            if game_name in {"Final Fantasy"}:
+                continue
+            multiworld = setup_default_world(world_type)
+            locations = Counter([location for location in multiworld.get_locations()])
+            if len(locations) >= 1:
```

Comment:
```suggestion
            locations = Counter(multiworld.get_locations())
            if locations:
```
didn't test, but pretty sure this is all not needed.

Reply:
about to check, almost did this, but i assumed that wouldn't work and just didn't lmao.